components/common/ColorModeButton.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div>
+    <a @click="toggleColorMode()" class="navbar-item">
+      <ColorScheme placeholder="...">
+        <!-- SUN -->
+        <svg
```

Comment:
may just use FA icons? https://fontawesome.com/v5/icons/moon?s=solid

Reply:
> may just use FA icons? https://fontawesome.com/v5/icons/moon?s=solid


I find the FA5 [sun icon](https://fontawesome.com/v5/icons/sun?s=solid) not very good :confused:

Meanwhile FA6 got [some options](https://fontawesome.com/v6/icons/brightness?s=solid) which can be nice